portfolio/src/main/java/webapp/WEB-INF/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Consider adding a lang="en" in this tag: https://www.w3.org/International/questions/qa-html-language-declarations

This is important for assistive technology to know the language of the content.